tests/unit/dataclass_strings_test.py

```
@@ -0,0 +1,373 @@
+        assert "0.0.123" in result
+        assert "symbol='TEST'" in result
+        assert "balance=1000" in result
+
+    def test_token_update_params_str(self):
+        """Test TokenUpdateParams string generation."""
+        from hiero_sdk_python.tokens.token_update_transaction import TokenUpdateParams
+        from hiero_sdk_python.account.account_id import AccountId
+        
+        params = TokenUpdateParams(
+            treasury_account_id=AccountId(0, 0, 456),
+            token_name="Updated Token"
```

Ruff (0.14.10)

Possible hardcoded password assigned to argument: "token_name"

(S106)